Documentation/SystemModules.md

```diff
@@ -98,7 +101,8 @@ First create a directory called `CJasPer` parallel to `CJPEG` and our example ap
     CJPEG$ cd ..
     $ mkdir CJasPer
     $ cd CJasPer
-    CJasPer$ touch module.modulemap Package.swift
+    CJasPer$ swift package init
```

Comment:
So isn't the real way to fix this to add "system-module" (or something like that) to the `init` subcommand's --type flag?